src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1147,6 +1148,8 @@ public enum DiskAccessMode
         mmap,
         mmap_index_only,
         standard,
+        legacy,
```

Comment:
note to self - we should validate if unsupported enums are not selected for sstables io